client/components/dialog/style.scss

```diff
@@ -113,6 +118,10 @@
 		margin-left: 0;
 		margin-right: 10px;
 	}
+
+	@include breakpoint( "<480px" ) {
+		margin: 2px 0;
```

Comment:
Ideally breakpoints are mobile first (`>480px` is the override).

Reply:
Oh well, while I agree, it's also worth to mention we haven't been very consistent with those. The number of `>` usages is not far from the number of `<` usages, so I'll pass in favor of simplicity.